src/com/richard/walker/weatherstation/utilities/WeatherGenerator.java

```diff
@@ -0,0 +1,35 @@
+package com.richard.walker.weatherstation.utilities;
+
+import com.richard.walker.weatherstation.domain.WeatherStation;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public final class WeatherGenerator {
+    private static final int MAX_TEMP = 110;
+    private static final int MIN_TEMP = 20;
+    private static final int MAX_WIND_SPEED = 30;
+    private static final int MAX_HUMIDITY = 100;
+
+    public static WeatherStation[] generateWeatherDataFor(String[] cities) {
+        int numCities = cities.length;
+
+        WeatherStation[] weatherStations = new WeatherStation[numCities];
+        ThreadLocalRandom rand = ThreadLocalRandom.current();
```

Comment:
Your intent with this class is to generate some random data for the weather station right? Is there a particular reason why you chose to use ```ThreadLocalRandom``` class over ```java.util.Random```?